lib/bulkPrefixer.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
Sorry, no I meant the `bulkPrefixer` in general.  I see that its concat-ing the index JSON onto the chunk, but what's the purpose?